index.d.ts

```diff
@@ -231,10 +231,79 @@ export interface LineSymbolizer extends BaseSymbolizer {
  */
 export type PointSymbolizer = IconSymbolizer | MarkSymbolizer | TextSymbolizer;
 
+/**
+ * A single entry for the ColorMap.
+ */
+export interface ColorMapEntry {
+  color: string;
+  quantity?: number;
+  label?: string;
+  opacity?: number;
+}
```

Comment:
All color map entries (at least for SLD, see [here](https://docs.geoserver.org/stable/en/user/styling/sld/reference/rastersymbolizer.html#cql-expressions)) allow for CQL expression. If we want to support this, we should support number and string inputs for all entries.

Reply:
Solved in https://github.com/terrestris/geostyler-style/pull/113/commits/62db216711613220b62772ad871f0e54b8728c9b

Reply:
Update: In coordination with @KaiVolland we decided to not support CQl expressions for now as they are geoserver specific and not part of the official standard. Reverted corresponding commit.